spot_rl_experiments/spot_rl/envs/place_env.py

```diff
@@ -68,13 +68,23 @@ def get_observations(self):
 class SpotSemanticPlaceEnv(SpotBaseEnv):
     """This is Spot semantic place class"""
 
-    def __init__(self, config, spot: Spot):
+    def __init__(self, config, spot: Spot, use_semantic_place: bool = False):
         # We set the initial arm joints
         config.INITIAL_ARM_JOINT_ANGLES = copy.deepcopy(
             config.INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE
         )
-        super().__init__(config, spot)
-
+        max_lin_dist_sem_place = (
+            "MAX_LIN_DIST_SEMANTIC_PLACE" if use_semantic_place else "MAX_LIN_DIST"
```

Comment:
do we need this if else case? wouldn't max_lin_dist always be `MAX_LIN_DIST_SEMANTIC_PLACE`?